packages/ab-testing/__tests__/functions.test.ts

```diff
@@ -0,0 +1,80 @@
+import { hashObject } from '@appannie/ab-testing-hash-object';
+import { validateAllocation, Cohort } from '../src';
+
+describe('test validateAllocation func', () => {
```

Comment:
can you also add tests to the react & python packages, like integration tests

Reply:
I'm not very familiar with Python but if no one is around to do it I can do that, it just may take a bit longer 😅

Reply:
@ZigZagT can we approach this separately?

Reply:
It's fine to attack it separately but we need a clear ETA

Otherwise, we're risking of leaving the python package behind forever.

Reply:
Going to resolve this in another PR. If we need a timeline, I can probably get this done by end of the week.

Reply:
Thank you for the effort! It's super important to keep the packages in a consistent manner.